src/devtools/client/debugger/src/components/PrimaryPanes/Outline.js

```diff
@@ -72,10 +72,16 @@ export class Outline extends Component {
       this.setFocus(this.props.cursorPosition);
     }
 
-    if (this.focusedElRef) {
-      if (!isVisible(this.focusedElRef, this.refs.outlineList)) {
-        this.focusedElRef.scrollIntoView({ block: "center" });
-      }
+    //confirm we aren't rescrolling back up the outline panel unnecessarily
+    const isUniqueEvent =
```

Comment:
does this need to be `JSON.stringify` or would `prevState.focusedItem !== this.state.focusedItem` work?